Hymns/Common/CustomColors.swift

```diff
@@ -0,0 +1,14 @@
+import SwiftUI
+
+//Colors use primarily for tags and are from Figma specs
+struct CustomColors {
+    static let backgroundBlue = Color(red: 2/255, green: 118/255, blue: 254/255, opacity: 0.2)
+    static let backgroundGreen = Color(red: 80/255, green: 227/255, blue: 194/255, opacity: 0.2)
+    static let backgroundYellow = Color(red: 255/255, green: 209/255, blue: 0/255, opacity: 0.2)
+    static let backgroundRed = Color(red: 255/255, green: 0, blue: 31/255, opacity: 0.2)
+
+    static let foregroundBlue = Color(red: 2/255, green: 118/255, blue: 254/255, opacity: 1.0)
+    static let foregroundGreen = Color(red: 35/255, green: 190/255, blue: 155/255, opacity: 1.0)
+    static let foregroundYellow = Color(red: 176/255, green: 146/255, blue: 7/255, opacity: 1.0)
+    static let foregroundRed = Color(red: 255/255, green: 0, blue: 31/255, opacity: 0.78)
+}
```

Comment:
I don't think this would be used outside of tags right? We should make these part of the TagColors enum. Just create a `backgroundColor` param and a `foregroundColor` param kind of like how I did the with `abbreviatedValue` in HymnType

Reply:
The tricky part is I also use them in TagSelectedLabelView. But in that case I need to be able to use them as a static since there is not an instantiated TagColor that is being switched on. But I also use these same colors as an instance property for TagColor so in that case they are not static. 

I did something similiar in my latest commit to what you did in HymnType. Lmk what you think. The problem with it is that it seemed to still add alot more code. Additionally, I can't use the enum function call within the stroke modifer in SelectLabelView so have to now hardcode the color there.  Also it introduced a wierd issue where the color is not being displayed correctly for SelectLabelView even though nothing changed for the colors. 
![Simulator Screen Shot - iPhone 8 - 2020-06-12 at 09 43 20](https://user-images.githubusercontent.com/15247726/84514857-4d93f900-ac91-11ea-8a97-eb486aaa72d1.png)

Reply:
I don't see anything called TagSelectedLabelView